src/components/node-list/node-list-items.js

```diff
@@ -176,14 +223,16 @@ export const getFilteredNodeItems = createSelector(
 
 /**
  * Get formatted list of sections
+ * @param {boolean} flag value of modularpipeline flag
  * @return {array} List of sections
  */
-export const getSections = createSelector(() =>
-  Object.keys(sidebar).map((name) => ({
+export const getSections = (flag) => {
```

Comment:
Might be clearer to name the `flag` as something like `modularPipelineFlag`?

Reply:
I have already elaborated that this refers to the modular pipeline flag in the comments, not sure of the need to repeat this in the naming and opt for a much longer variable name?

Reply:
For sure it's good that it's documented, but yeah I also think in this case a more verbose name is a good idea, since it's a specific flag the function is looking for.